doc/man3/SSL_read_early_data.pod

```diff
@@ -181,7 +181,7 @@ with then the lower of the two values will apply.
 The whole purpose of early data is to enable a client to start sending data to
 the server before a full round trip of network traffic has occurred. Application
 developers should ensure they consider optimisation of the underlying TCP socket
-to obtain a performant solution. For example Nagle's algorithm is commonly used
+to obtain an efficient solution. For example Nagle's algorithm is commonly used
```

Comment:
This one seems like a matter of taste, but I don't really prefer one over the other, myself.

Reply:
'performant' is missing in a some dictionnaries